pgoapi/rpc_api.py

```diff
@@ -53,10 +53,10 @@
 
 class RpcApi:
 
-    RPC_ID = 0
-    START_TIME = 0
-
     def __init__(self, auth_provider, device_info):
+        self.RPC_REQUEST = 1
```

Comment:
Moving the fields into the class seems problematic because for every request there is a new RpcApi object, so each and every request will yield the same request id in the end?